ui/jstests/test-learning-resource.jsx

```diff
@@ -554,8 +561,60 @@ define(['QUnit', 'jquery', 'react', 'lodash', 'learning_resources',
     function(assert) {
       var div = document.createElement("div");
       assert.ok($(div).html().length === 0);
-      LearningResources.loader("repo", "1", function() {}, function() {}, div);
+      LearningResources.loader(
+        "repo", "1", function() {}, function() {}, function() {}, false,
+        false, div
+      );
       assert.ok($(div).html().length > 0);
     }
   );
+
+  QUnit.test('Assert learning resource panel does close',
+    function (assert) {
+      var done = assert.async();
+      var lrMountCount = 0;
+      var closeLearningResourcePanelCount = 0;
+      var closeLearningResourcePanel = function() {
+        closeLearningResourcePanelCount += 1;
+      };
+
+      var Container = React.createClass({
+        getInitialState: function() {
+          return {closeLRPanel: false};
+        },
+        render: function() {
+          lrMountCount += 1;
+          return <LearningResourcePanel
+            repoSlug="repo"z
+            ref={this.afterMount}
+            closeLearningResourcePanel={this.props.closeLearningResourcePanel}
+            closeLRPanel={this.state.closeLRPanel}
+            learningResourceId="1" />;
+        },
+        afterMount: function () {
+          waitForAjax(9, function () {
```

Comment:
when closeLRPanel = false then three calls.
when closeLRPanel = true then three calls on componentDidMount and three on closing LR panel 
```componentWillReceiveProps```